### DISCUSSION TITLE
feature request: jira issue sprint

### DISCUSSION THREAD

**Discussion by realtime-neil** [ankitpokhrel/jira-cli#183] [Ideas] [answered]
> Please consider [opening a discussion](https://github.com/ankitpokhrel/jira-cli/discussions/categories/ideas) in `idea` category for the proposed feature if applicable. 

**Is your feature request related to a problem? Please describe.**
I have sprints. I have issues. I would like a way to associate the latter with the former.

**Describe the solution you'd like**
`jira issue sprint ISSUE-KEY SPRINT`

**Describe alternatives you've considered**
Using a web browser, but that sucks because Jira web pages suck.

**Additional context**
I don't know how common the "Sprint" attribute is for your "normal" Jira configurations, but my personal anecdata suggests that it's overwhelmingly popular.

**Comment by ankitpokhrel** [comment]
If I get it right, you want to assign sprint to an issue. It's something that is on my radar and hopefully will work on it soon. I am currently doing it from the UI as well.

**Comment by realtime-neil** [comment]
@ankitpokhrel yes, you understand perfectly. Thank you for taking the time to think about this; I do appreciate it.

**Comment by ankitpokhrel** [comment, marked as answer]
This is done in #205 and will be available in the next release.

```sh
$ jira sprint add -h
Add issues to sprint.

USAGE
  jira sprint add SPRINT_ID ISSUE-1 [...ISSUE-N] [flags]

FLAGS
  -h, --help   help for add

INHERITED FLAGS
  -c, --config string    Config file (default is $HOME/.config/.jira/.config.yml)
      --debug            Turn on debug output
  -p, --project string   Jira project to look into (defaults to $HOME/.config/.jira/.config.yml)

ARGUMENTS
  SPRINT_ID		ID of the sprint on which you want to assign issues to, eg: 123
  ISSUE-1 [...ISSUE-N]	Key of the issues to add to the sprint (max 50 issues at once)

EXAMPLES
  $ jira sprint add SPRINT_ID ISSUE-1 ISSUE-2

ALIASES
  assign

LEARN MORE
  Use 'jira <command> <subcommand> --help' for more information about a command.

```

More on [readme](https://github.com/ankitpokhrel/jira-cli#add-2). You can use `jira sprint list` to get the sprint id.

**Reply by realtime-neil** [reply]
Nice! Thanks @ankitpokhrel
